packages/clients/meldemichel/example/report.html

```diff
@@ -53,12 +53,12 @@
     <div class="mock-content">
       <p>This is a minimal test page for the REPORT environment.</p>
       <div id="polarstern"></div>
-      <p>Communicated data:</p>
-      <p>Coordinates: <span id="test-display-coordinate">null</span></p>
-      <p>Address: <span id="test-display-address">null</span></p>
-      <p>Address data: <span id="test-display-address-data">null</span></p>
+      <h2>Subscribed data</h2>
+      <ul id="subscribed-data-list"></ul>
+      <h2>Setting data</h2>
+      <p>This uses test data. See report.html contents.</p>
       <button onclick="_polar.setMapState()">
-        Set parameters after rendering
+        Set test parameters after rendering (only works initially)
```

Comment:
Clicking the button doesn't center on the respective coordinate which is irritating.  
Was that the case before as well?

Reply:
It centers on the `mapCenter`, not the `vendor_maps_position`, which is the pin coordinate. This is intended: This function aims to recreate a previous user view.